libs/framework/gtest/src/DependencyManagerTestSuite.cc

```diff
@@ -486,6 +487,234 @@ TEST_F(DependencyManagerTestSuite, RequiredDepsAreInjectedDuringStartStop) {
     celix_bundleContext_unregisterService(dm.bundleContext(), svcId);
 }
 
+TEST_F(DependencyManagerTestSuite, RemoveOwnDependencyShouldNotLeadToDoubleStop) {
+    //Given a component LifecycleComponent which provides and requires a TestService service
```

Comment:
I really like these kind of comments! They increase the readability of the tests tremendously.